### ISSUE TITLE
CVE-2020-1472.ps1: if-else formatting: 'else' cannot appear on its own line in PS 5.1

### ISSUE THREAD

**Issue by mfoust** [cisagov/cyber.dhs.gov#185]
https://github.com/cisagov/cyber.dhs.gov/blob/4125c2bbc60818e7fc82db1620a2d473c2217aab/assets/report/ed-20-04_script/CVE-2020-1472.ps1#L22

OS: Windows Server 2016
Powershell version: 5.1

}
else
{

will produce the error

> else : The term 'else' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

It's an easy fix. It just needs to be changed to

} else
{

or

} else {

(Edit: GitHub removed line breaks that are necessary for identifying this problem.)

**Comment by codaamok** [comment]
This is not true. It is possible you have copied and pasted the script in to your console, rather than invoking from script file, which would cause that error message.

**Comment by mfoust** [comment]
> It is possible you have copied and pasted the script in to your console...

Yes. On my team, we test all scripts in interactive shells in a test environment first and bugfix there before saving them to ps1s. We have various off-topic reasons for doing it that way - but I've found it's a pretty common workflow across organizations.

I honestly didn't know that uncuddled elses were interpreted differently in ps1s than in interactive shells until you pointed it out. I thought that Powershell (at least 5.1, which is what we run) required the brace style I described above. So thank you, @codaamok for teaching me something about Powershell.
